simplq/src/store/asyncActions/pauseQueue.js

```diff
@@ -0,0 +1,32 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { useMakeAuthedRequest } from 'api/auth';
+import * as RequestFactory from 'api/requestFactory';
+import { useHistory } from 'react-router';
+
+const typePrefix = 'pauseQueue/action';
+
+/**
+ * A hook to access the pauseQueue async action creator.
+ *
+ * @returns — pauseQueue async action creator
+ */
+const usePauseQueue = () => {
+  const makeAuthedRequest = useMakeAuthedRequest();
```

Comment:
Maybe this should be renamed to useSetQueueStatus, because it's also used to set the status to ACTIVE and not just to pause a queue. What do you say @daltonfury42?

Reply:
I renamed the file to `updateQueueStatus` and the function to `useUpdateQueueStatus`